apps/web/src/pages/api/affiliates-program/affiliate-claim-fee.ts

```diff
@@ -0,0 +1,31 @@
+import { getCookie } from 'cookies-next'
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { AFFILIATE_SID } from 'pages/api/affiliates-program/affiliate-login'
+
+const affiliateClaimFee = async (req: NextApiRequest, res: NextApiResponse) => {
+  const cookie = getCookie(AFFILIATE_SID, { req, res, sameSite: true })
+
+  if (!process.env.AFFILIATE_PROGRAM_API_URL && req.method === 'POST' && !cookie) {
+    return res.status(400).json({ message: 'API URL Empty' })
+  }
+
+  const requestUrl = `${process.env.AFFILIATE_PROGRAM_API_URL}/affiliate/fee/claim/request`
+  const response = await fetch(requestUrl, {
```

Comment:
Why do we need to proxy the request instead of sending the request to BE from FE directly?

Reply:
We don't want others to be able to call directly BE API

Reply:
actually can use rewrite if nothing change in the middle

Reply:
hmm I still don't get what's the difference. Others can simply call the fe api endpoint instead? It will still end up reaching our BE. What is the benefit? We can have an extra layer of parameter validation?